src/GraphQL.Conventions/GraphQL.Conventions.csproj

```diff
@@ -2,8 +2,8 @@
 
   <PropertyGroup>
     <Description>GraphQL Conventions for .NET</Description>
-    <VersionPrefix>7.0.0</VersionPrefix>
-    <PackageVersion>7.0.0</PackageVersion>
+    <VersionPrefix>7.1.0</VersionPrefix>
+    <PackageVersion>7.1.0</PackageVersion>
     <Authors>Tommy Lillehagen</Authors>
     <TargetFrameworks>net6.0;netstandard2.0</TargetFrameworks>
     <LangVersion>9.0</LangVersion>
```

Comment:
I would change to latest in dirbuildprops + also remove DepsFileGenerationMode, AssemblyName, PackageId, GeneratePackageOnBuild, both DefineConstants + change GenerateDocumentationFile to true and DebugType to embedded.

Reply:
Those changes should be made in a separate PR.

Reply:
At least, I'd prefer to make those changes in a separate PR where they can be independently reviewed.  This PR really is just trying to get the basic workflows in place to switch 1:1 to GitHub Actions from Appveyor, and to get the test projects usable -- which probably should have been two PRs.

Reply:
Ok